files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js

```diff
@@ -327,6 +327,26 @@ class ApplicationContextMenuItem extends PopupMenu.PopupBaseMenuItem {
 
     activate (event) {
         switch (this._action) {
+            case "show_launcher_file":
+                let fileapp = Gio.app_info_get_default_for_type('inode/directory', true);
+                if (fileapp) {
+                    if (fileapp.get_id() == 'nemo.desktop') {
+                        Util.spawnCommandLine('nemo --no-desktop "' + this._appButton.app.get_app_info().get_filename() + '"');
```

Comment:
I forgot to mention that the `--no-desktop` CLI option might not be needed. Nemo (the process) no longer handles the desktop (the `nemo-desktop` process does it now). The `--no-desktop` CLI option isn't used by the `nemo` process and it doesn't trigger a parse error because the option was added as a *"dummy"* option apparently for retro-compatibility.

Reply:
Thank you for the info.

Is there any chance that a newer version of cinnamon could be running with an older version of nemo that needs the option?

Reply:
> Is there any chance that a newer version of cinnamon could be running with an older version of nemo that needs the option?

I hardly doubt it. The `--no-desktop` CLI option was removed around Nemo 3.4.x (about 6 versions/3 years ago). I don't think that anyone using Cinnamon 4.6.x with a Nemo version older than 3.4.x will have a usable system nor if they could even install those versions together at all.